demisto_sdk/commands/common/constants.py

```diff
@@ -344,6 +344,13 @@ class FileType(str, Enum):
     FileType.GENERIC_TYPE,
     FileType.GENERIC_FIELD,
 ]
+XSIAM_ENTITIES = [
```

Comment:
How is this different than `SIEM_ONLY_ENTITIES`?

Reply:
looked for  consts with the word `XSIAM`, i was sure there must be a list, anyways, changing to this